llvm/include/llvm/CodeGen/ReachingDefAnalysis.h

```diff
@@ -138,6 +141,9 @@ class ReachingDefAnalysis : public MachineFunctionPass {
   DenseMap<MachineInstr *, int> InstIds;
 
   MBBReachingDefsInfo MBBReachingDefs;
+  using MBBFrameObjsReachingDefsInfo =
```

Comment:
It would be nice to leave a comment describing this data structure.

Reply:
done